app/Mage.php

```diff
@@ -48,6 +48,14 @@
 
 Varien_Autoload::register();
 
+/** AUTOLOADER PATCH **/
+if (file_exists($autoloaderPath = BP . DS . 'vendor/autoload.php') ||
+    file_exists($autoloaderPath = BP . DS . '../vendor/autoload.php')
+) {
+    require $autoloaderPath;
+}
+/** AUTOLOADER PATCH **/
+
```

Comment:
Is this hunk of code related to this PR?

Reply:
absolutely no, my fault, sorry and thanks for seeing it!